wordpress-mobile/WordPress-Android
# [Blogging prompts v1] Add "Learn more" to "My Site" prompt card

Fixes #

To test:

- Enable blogging prompts feature flag;
- Tap on "My Site" in bottom navigation;
- Tap on "HOME" tab;
- Scroll to prompt card;
- Tap on overflow menu;
- Tap on "Learn more" button. Blogging prompts introduction should be shown.

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing and updated unit tests

3. What automated tests I added (or what prevented me from doing so)
Updated `BloggingPromptsCardAnalyticsTrackerTest.kt`

PR submission checklist:
